internal/provider/file/file_test.go

```diff
@@ -10,6 +10,10 @@ import (
 )
 
 var (
+	defaultImageDefaults = &model.Image{
```

Comment:
Related to my previous comment this might it:

```suggestion
	defaultImageDefaults = model.Image{
```